clang/docs/ReleaseNotes.rst

```diff
@@ -160,6 +160,9 @@ Bug Fixes in This Version
 Bug Fixes to Compiler Builtins
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- ``__is_layout_compatible`` no longer requires the empty bases to be the same in two
+  standard-layout classes. It now only compares non-static data members.
```

Comment:
Hmm.  I was under the impression that this feature had not been previously released; is that incorrect?

Reply:
That is correct, but the feature made into the 19 branch, whereas this fix did not (I hope we can backport it), hence a release note.

Reply:
Is the plan to remove the release note from LLVM 20 if we're able to backport the fix?

Reply:
I'm not sure, but this indeed is a possible course of action.
If you have preferences how this should be handled, let us know.
Personally I'm more focused on getting this backported.

Reply:
We should backport these changes, drop the Clang 20 release note, and no need for a Clang 19.x release note to be added.